src/main/java/net/dancier/kikeriki/KikerikiApplication.java

```diff
@@ -28,12 +30,16 @@ public class KikerikiApplication implements ApplicationRunner
   @Autowired
   ConfigurableApplicationContext context;
 
+  @Autowired
+  GitProperties gitProperties;
+
   ListenableFuture<Integer> consumerJob;
 
   @Override
   public void run(ApplicationArguments args)
   {
     log.info("Starting KikerikiConsumer");
+    MDC.put("commit", gitProperties.getShortCommitId());
     consumerJob = taskExecutor.submitListenable(kikerikiConsumer);
```

Comment:
putting the commit id in the logs will make it easy to find out if we have problems cause one overdeployed the other one ;-)

Reply:
No offense.
But unnecessary, since I activated the acturator and throwed in the git-commit-id-plugin.
Just browse to HOST:PORT/CONTEXTPATH/actuator/info